GVFS/GVFS.UnitTests/Service/RepoRegistryTests.cs

```diff
@@ -0,0 +1,200 @@
+﻿using GVFS.Common;

```

Comment:
This is the same old set of tests that were there in `GVFS.UnitTests.Windows` project. Now they can be run on the Mac as well. So I moved it to `GVFS.UnitTests/Service` directory.